modules/cli/src/main/scala/scala/cli/commands/directories/Directories.scala

```diff
@@ -21,7 +21,7 @@ object Directories extends ScalaCommand[DirectoriesOptions] {
       sys.exit(1)
     }
 
-    val directories = options.directories.directories
+    val directories = scala.build.Directories.directories
```

Comment:
NIT just import it here
```suggestion
    val directories = Directories.directories
```

Reply:
It doesn't work, but this class is named `Directories`.